app/views/includes/no-results.nunjucks

```diff
@@ -3,10 +3,11 @@
     <h2 class="results__header results__header--none">There are no pharmacies within 20 miles of {{ location }}</h2>
     <div class="results__none-content">
       <p>This service only provides information about pharmacies in England.</p>
-      <p>You might be getting this result because you live in Scotland, Wales or Northern Ireland, or because you live near a border with one of those countries.</p>
-      <p><a href="http://www.nhs24.com/findlocal/">Find pharmacies in Scotland on the NHS 24 website.</a></p>
-      <p><a href="http://www.nhsdirect.wales.nhs.uk/localservices/searchlocalservices.aspx?s=Pharmacy">Find pharmacies in Wales on the NHS Direct Wales website.</a></p>
-      <p><a href="http://servicefinder.hscni.net">Find pharmacies in Northern Ireland on the Health and Social Care Online website.</a></p>
+      <p>You might be getting this result because you live in Scotland, Wales or Northern Ireland, or because you live near a border with one of those countries. It could also be because you live on the Isle of Man.</p>
```

Comment:
No wording changes will be made as part of this change. For a future iteration...